Widgets/Animation/VarmintWidgetAnimationCollection.cs

```diff
@@ -4,13 +4,29 @@
 
 namespace MonoVarmint.Widgets
 {
-    //--------------------------------------------------------------------------------------
-    /// <summary>
-    /// MoveOffsetLinear - animate an offset from one value to another
-    /// </summary>
-    //--------------------------------------------------------------------------------------
     public partial class VarmintWidgetAnimation
     {
+        //--------------------------------------------------------------------------------------
+        /// <summary>
+        /// CycleFade - Fades between a start and end opacity in a sinusoidal manner
+        /// </summary>
+        //--------------------------------------------------------------------------------------
+        public static VarmintWidgetAnimation CycleFade(
+            double durationSeconds,
+            float startOpacity,
+            float endOpacity,
+            float cycleTime
+            )
+        {
+            var opacityChange = startOpacity - endOpacity;
```

Comment:
nit:  This is a little confusing:   If the start opacity is 0 and the end opacity is 1, I would expect the opacity change to be 1, not -1.